### PR TITLE
[WebPubSub] Release 1.1.0 for bug fix

### PR DESCRIPTION
**Pull request by xingsy97**
# Description
- Use the correct REST API parameter name `groups` in method `get_client_access_token`

autorest command：`autorest --version=3.9.2 --use=@autorest/python@6.1.11 --use=@autorest/modelerfour@4.24.3 ./swagger/README.md`
# All SDK Contribution checklist:
- [ ] **The pull request does not introduce [breaking changes]**
- [ ] **CHANGELOG is updated for new features, bug fixes or other significant changes.**
- [ ] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-python/blob/main/CONTRIBUTING.md).**

## General Guidelines and Best Practices
- [ ] Title of the pull request is clear and informative.
- [ ] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-python/blob/main/CONTRIBUTING.md##building-and-testing)
- [ ] Pull request includes test coverage for the included changes.

### CODE REVIEW

File: sdk/webpubsub/azure-messaging-webpubsubservice/CHANGELOG.md [hunk 1]
```diff
@@ -1,14 +1,9 @@
 # Release History
 
-## 1.1.0b2 (Unreleased)
-
-### Features Added
-
-### Breaking Changes
+## 1.1.0b2
```
Comment (vicancy) [inline sdk/webpubsub/azure-messaging-webpubsubservice/CHANGELOG.md:3]: let's release a stable version  `1.1.0`

File: sdk/webpubsub/azure-messaging-webpubsubservice/azure/messaging/webpubsubservice/_operations/_operations.py [hunk 2]
```diff
@@ -705,8 +705,8 @@ def get_client_access_token(
         :paramtype roles: list[str]
         :keyword minutes_to_expire: The expire time of the generated token. Default value is 60.
         :paramtype minutes_to_expire: int
-        :keyword group: Groups that the connection will join when it connects. Default value is None.
-        :paramtype group: list[str]
+        :keyword groups: Groups that the connection will join when it connects. Default value is None.
```
Comment (vicancy) [inline sdk/webpubsub/azure-messaging-webpubsubservice/azure/messaging/webpubsubservice/_operations/_operations.py:708]: also add the groups description to the mixin API

File: sdk/webpubsub/azure-messaging-webpubsubservice/CHANGELOG.md [hunk 3]
```diff
@@ -1,14 +1,9 @@
...
-
-### Features Added
-
-### Breaking Changes
+## 1.1.0
 
 ### Bugs Fixed
-
-### Other Changes
+- Use the correct REST API parameter name `groups` in method `get_client_access_token`
```
Comment (vicancy) [inline sdk/webpubsub/azure-messaging-webpubsubservice/CHANGELOG.md:6]: also mention the jwt version upgrade?

File: sdk/webpubsub/azure-messaging-webpubsubservice/azure/messaging/webpubsubservice/aio/_operations/_operations.py [hunk 4]
```diff
@@ -121,7 +121,7 @@ async def get_client_access_token(
         user_id: Optional[str] = None,
         roles: Optional[List[str]] = None,
         minutes_to_expire: int = 60,
-        group: Optional[List[str]] = None,
+        groups: Optional[List[str]] = None,
```
Comment (vicancy) [inline sdk/webpubsub/azure-messaging-webpubsubservice/azure/messaging/webpubsubservice/aio/_operations/_operations.py:124]: possible to add some test?

File: sdk/webpubsub/azure-messaging-webpubsubservice/setup.py [hunk 5]
```diff
@@ -64,7 +64,7 @@
     install_requires=[
         "isodate<1.0.0,>=0.6.1",
         "azure-core<2.0.0,>=1.24.0",
-        "pyjwt>=1.7.1",
+        "pyjwt>=2.0.0"
     ],
     python_requires=">=3.7",
```
Comment (msyyc) [inline sdk/webpubsub/azure-messaging-webpubsubservice/setup.py:69]:
```suggestion
    python_requires=">=3.8",
```
Python SDK doesn't support 3.7 anymore

File: sdk/webpubsub/azure-messaging-webpubsubservice/setup.py [hunk 6]
```diff
@@ -64,7 +64,7 @@
     install_requires=[
         "isodate<1.0.0,>=0.6.1",
         "azure-core<2.0.0,>=1.24.0",
-        "pyjwt>=1.7.1",
+        "pyjwt>=2.0.0"
```
Comment (msyyc) [inline sdk/webpubsub/azure-messaging-webpubsubservice/setup.py:67]: Please update setup.py with reference of https://github.com/Azure/azure-sdk-for-python/blob/7181b93ce68d17189dbe4f054449d25ace3f523c/tools/azure-sdk-tools/packaging_tools/templates/packaging_files/setup.py#L53-L61

File: sdk/webpubsub/azure-messaging-webpubsubservice/swagger/README.md [hunk 7]
```diff
@@ -67,6 +67,10 @@ directive:
     where: $["paths"]["/api/hubs/{hub}/:generateToken"].post.parameters
     transform: >
         $[2]["x-ms-client-name"] = "roles"
+  - from: swagger-document
+    where: $["paths"]["/api/hubs/{hub}/:generateToken"].post.parameters
+    transform: >
+        $[5]["x-ms-client-name"] = "groups"
```
Comment (msyyc) [inline sdk/webpubsub/azure-messaging-webpubsubservice/swagger/README.md:73]: If possible, it is better to re-enable the test https://github.com/Azure/azure-sdk-for-python/tree/1bf8aaa72dbc358f2fb9125d045a32d87499e5c0/sdk/webpubsub/azure-messaging-webpubsubservice/tests. These test could be activated after migration to test proxy: https://github.com/Azure/azure-sdk-for-python/blob/1bf8aaa72dbc358f2fb9125d045a32d87499e5c0/sdk/webpubsub/azure-messaging-webpubsubservice/tests/STATE_OF_TESTS.md

File: sdk/webpubsub/azure-messaging-webpubsubservice/CHANGELOG.md [hunk 8]
```diff
@@ -1,14 +1,9 @@
 # Release History
 
-## 1.1.0b2 (Unreleased)
-
-### Features Added
-
-### Breaking Changes
+## 1.1.0
```
Comment (msyyc) [inline sdk/webpubsub/azure-messaging-webpubsubservice/CHANGELOG.md:3]:
```suggestion
## 1.1.0 (2024-02-28)
```

File: sdk/webpubsub/azure-messaging-webpubsubservice/swagger/README.md (hunk 7 above)
Comment (xingsy97) [reply sdk/webpubsub/azure-messaging-webpubsubservice/swagger/README.md:73]: let's do this in another pr

File: sdk/webpubsub/azure-messaging-webpubsubservice/setup.py (hunk 6 above)
Comment (xingsy97) [reply sdk/webpubsub/azure-messaging-webpubsubservice/setup.py:65]: removed some classifier

File: sdk/webpubsub/azure-messaging-webpubsubservice/azure/messaging/webpubsubservice/aio/_operations/_patch.py [hunk 9]
```diff
@@ -229,7 +227,6 @@ async def send_to_group(  # pylint: disable=inconsistent-return-statements
                 "The content_type '{}' is not one of the allowed values: "
                 "['application/json', 'application/octet-stream', 'text/plain']".format(content_type)
             )
-
```
Comment (msyyc) [inline sdk/webpubsub/azure-messaging-webpubsubservice/azure/messaging/webpubsubservice/aio/_operations/_patch.py:232]: https://dev.azure.com/azure-sdk/29ec6040-b234-4e31-b139-33dc4287b756/_apis/build/builds/3536542/logs/356 There are some mypy error in patch and samples. Better to fix it

Comment (xingsy97) [reply sdk/webpubsub/azure-messaging-webpubsubservice/azure/messaging/webpubsubservice/aio/_operations/_patch.py:232]: disabled mypy temporally

**Review by msyyc — APPROVED** [review]

**Pull request merged**